### PR TITLE
Release 0.4.1

### PR DESCRIPTION
**Pull request by ilanbiala**
* Test against Node 4.0.0
* Make c.log internal
* Add caching and containerized setup to Travis
* Update dependencies and code and remove unnecessary dependencies

### CODE REVIEW

File: app/log.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+log.white = function(value) {
+  console.log(chalk.white(value));
+};
+
+log.gray = function(value) {
+  console.log(chalk.gray(value));
+};
+
+module.exports = log;
```
Comment (codydaig) [inline app/log.js]: New Line

Comment (ilanbiala) [inline app/log.js]: @codydaig no joke, on my computer I have a newline, but somehow GitHub/Sourcetree do not want to commit it.

Comment (ilanbiala) [inline app/log.js]: @codydaig also, is there any way to add JSHint checking into our tests without adding something really complex like Grunt or Gulp?

Comment (codydaig) [inline app/log.js]: @ilanbiala I think so, I'll look into it.

**Comment by codydaig** [comment]
Initially, LGTM. I'm gonna pull down and test later to double check.

**Comment by ilanbiala** [comment]
@codydaig any issues when testing it?

**Comment by codydaig** [comment]
Tested and LGTM. :)

**Comment by lirantal** [comment]
Great stuff guys!

**Comment by codydaig** [comment]
When will npm be updated with the latest version?

**Comment by ilanbiala** [comment]
@codydaig I need to get collab status from @amoshaviv, but he hasn't responded yet. Whenever he does, I'll publish it.

**Comment by codydaig** [comment]
@ilanbiala Ah. Gotcha.

**Pull request merged**